src/main/java/duke/exception/ManipulateException.java

```diff
@@ -0,0 +1,21 @@
+package duke.exception;
+
+import duke.Ui;
+
+/**
+ * Represents an exception thrown when the user enters an invalid
+ * manipulate command keyword.
+ */
+public class ManipulateException extends DukeException {
```

Comment:
Perhaps you can include more explanation on what does "manipulate command keyword" mean?

Reply:
Seconded!